playbooks/adhoc/upgrades/upgrade.yml

```diff
@@ -226,7 +226,7 @@
     when: inventory_hostname != groups.masters.0
 
   - name: Restart master services
-    service: name="{{ openshift.common.service_type}}-master" state=restarted
+    service: name="{{ openshift.common.service_type}}-master" enabled=yes state=restarted
```

Comment:
Hmm, We do probably need to ensure that the new service is enabled since we are changing the service name, but we should do that through a different step that has state=started and enabled=yes.

I don't think we should be modifying enabled on restart actions in general.